backend/tests/unit/components/permissionHelpers.spec.js

```diff
@@ -0,0 +1,121 @@
+const config = require('config');
+const log = require('npmlog');
+
+const permissionHelpers = require('../../../src/components/permissionHelpers');
+
+log.level = config.get('server.logLevel');
+
+describe('filterAppAcronymRoles', () => {
+  it('should return the filtered acronym list', () => {
+    const roles = ['offline_access', 'uma_authorization', 'WEBADE_CFG_READ', 'WEBADE_CFG_READ_ALL', 'DOMO', 'MSSC'];
+    const result = permissionHelpers.filterAppAcronymRoles(roles);
+
+    expect(result).toBeTruthy();
+    expect(result).toHaveLength(2);
+  });
+
+  it('should handle an empty array', () => {
+    const roles = [];
+    const result = permissionHelpers.filterAppAcronymRoles(roles);
+    expect(result).toBeTruthy();
+    expect(result).toHaveLength(0);
+  });
+});
+
+const configForm = require('./fixtures/configForm.json');
+const sampleToken = require('./fixtures/token.json');
+const acronymDetail = require('./fixtures/acronymDetail.json');
```

Comment:
This should probably be declared after line 7 if we are doing it at this file level scope - otherwise bring it down to the describe function where it'd be used.

Reply:
Moved fixture declarations to top, as they are used by multiple describes